code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -550,7 +550,7 @@
 	deactivate(close_ui = TRUE)
 
 /// Set the initial color of the eyes on insert to be the mob's previous eye color.
-/obj/item/organ/eyes/robotic/glow/mob_insert(mob/living/carbon/eye_recipient, special = FALSE, movement_flags = DELETE_IF_REPLACED)
+/obj/item/organ/eyes/robotic/glow/on_mob_insert(mob/living/carbon/eye_recipient, special = FALSE, movement_flags)
```

Comment:
In some places, you have defined `on_mob_insert` twice: 

553: /obj/item/organ/eyes/robotic/glow/on_mob_insert
559: /obj/item/organ/eyes/robotic/glow/on_mob_insert